astropy/coordinates/tests/test_sites.py

```diff
@@ -56,17 +48,17 @@ def test_online_stes():
 # this will *try* the online so we have to make it remote_data, even though it
 # falls back on the non-remote version
 def test_EarthLocation_basic():
-    keckel = EarthLocation.of_site('keck')
-    lon, lat, el = keckel.to_geodetic()
-    assert_quantity_allclose(lon, -1*Longitude('155:28.7', unit=u.deg),
-                             atol=0.001*u.deg)
-    assert_quantity_allclose(lat, Latitude('19:49.7', unit=u.deg),
+    greenwichel = EarthLocation.of_site('greenwich')
+    lon, lat, el = greenwichel.to_geodetic()
+    assert_quantity_allclose(lon, -1*Longitude('51:28:30', unit=u.deg),
                              atol=0.001*u.deg)
-    assert_quantity_allclose(el, 4160*u.m, atol=1*u.m)
+    assert_quantity_allclose(lat, Latitude('0:0:0', unit=u.deg),
```

Comment:
Oh, duh, good point.  Will fix that (it was skipped because it's remote_data...)